src/Handlebars/Context.php

```diff
@@ -73,6 +95,18 @@ public function pushIndex($index)
         array_push($this->index, $index);
     }
 
+    /**
+     * Pushes data variables onto the stack. This is used to support @data variables.
+     * @param array $data Associative array where key is the name of the @data variable and value is the value.
+     */
+    public function pushData($data)
+    {
+        if (!$this->enableDataVariables) {
+            throw new LogicException('data variables are not supported due to the enableDataVariables configuration. Remove the call to data variables or change the setting.');
```

Comment:
Nitpicky but could the D in data be capitalized? Same with all of the other error message strings.